test/integration/tests/button/validation.js

```diff
@@ -852,17 +852,27 @@ const buttonConfigs = [
         height: 20,
         valid: false,
       },
+      {
+        disableMaxHeight: true,
+        height: 20,
+        valid: false,
+      },
+      {
+        disableMaxHeight: true,
+        height: 60,
+        valid: true,
+      },
 
       // $FlowFixMe
-    ].map(({ height, size, valid }) => ({
+    ].map(({ height, valid, disableMaxHeight }) => ({
```

Comment:
It didn't seem like size was used at all in this sections so I removed it and used the sentence structure to indicate if the `disbaleMaxHeight` feature was enabled.